tests/st/utils/utils.py

```diff
@@ -48,7 +48,7 @@
 IPV6_RE = re.compile(r'inet6 ([a-fA-F\d:]+)/\d{1,3}')
 
 
-def calicoctl(command):
+def calicoctl(input, command):
```

Comment:
I'm not sure I see the point of this input parameter.  It makes reading the code a lot less clear and it's really just part of the main command.

Perhaps better to just rename this to `executeCommand()`, or perhaps `commandWithDatastoreEnvs()` if you really wanna describe what it's doing.

Reply:
As discussed, I misread this code so, comment above is not sensible.  However, we discussed an improved method signature for this that'll make the command more useful and extensible.  

```
type CalicoctlOptions struct {
	Filename string
	InsertFileAsStdin bool
}

func calicoctl(command string, options... Options) {
	do stuff.  options will be length or 1
}

calicoctl("delete tea pgtips")
calicoctl("create", Options{Filename: "input.yaml"})

```

Reply:
Perhaps this should live in a calicoctl sub-module, so you don't need so much repetition.

e.g.

```
calicoctl.Command("apply", Options{Filename: "input.yaml"})
```

Not too fussed though.

Reply:
oh silly me.  It's python

Use named parameters.